apps/transport/lib/jobs/consolidate_bnlc_job.ex

```diff
@@ -72,13 +79,20 @@ defmodule Transport.Jobs.ConsolidateBNLCJob do
       |> send_email_recap(%{
         dataset_errors: dataset_errors,
         validation_errors: validation_errors,
-        download_errors: download_errors
+        download_errors: Enum.filter(download_or_decode_errors, &match?({:download, _}, &1)),
+        decode_errors: Enum.filter(download_or_decode_errors, &match?({:decode, _}, &1))
       })
 
       :ok
     end
   end
 
+  defp validator_unavailable?(validation_errors) do
+    validation_errors
+    |> Enum.filter(&match?({:validator_unavailable_error, _, _}, &1))
+    |> Enum.any?()
+  end
```

Comment:
C'est du [boy scout](https://biratkirat.medium.com/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385) ceci et le renommage de `:validation_error` vers `:validator_unavailable_error`